inductiva/molecules/protein_solvation/README.md

```diff
@@ -38,25 +38,27 @@ To test this scenario we have available one example of a protein PDB file - [dow
 
 inductiva.api_key = "YOUR_API_KEY"
 
- # Initialize the scenario
- scenario = inductiva.molecules.ProteinSolvation(
-     protein_pdb = "alanine.pdb",
+# Initialize the scenario
+lysozyme = inductiva.molecules.examples.load_lysozyme()
+scenario = inductiva.molecules.ProteinSolvation(
+     protein_pdb = lysozyme,
      temperature = 300)
 
- # Run a simulation
- task = scenario.simulate(simulation_time_ns = 0.5)
+# Run a simulation
+task = scenario.simulate(simulation_time_ns = 0.01)
 
- # Get the simulation output on your local machine.
- output = task.get_output()
+# Get the simulation output on your local machine.
+output = task.get_output()
  ```
+This may take up to 2 minutes, depending on which machine you use.  
 The last code line downloads the files necessary to render and post-process our simulation.
 
 ## Output and Post-processing 
 The simulation output folder contains trajectory data spanning the simulation duration and the protein's topology. These files are necessary to generate an interactive trajectory visualization using [NGLview](https://github.com/nglviewer/nglview). Users have the flexibility to select the protein's representation type, including options to display or hide the backbone, and they can also specify the criteria for visualization selection. You can refer to the [documentation](https://nglviewer.org/ngl/api/manual/usage/selection-language.html) for guidance on creating custom selections according to your preferences.
 
 ```python
- output = task.get_output()
- output.render_interactive(representation="ribbon", add_backbone=True, selection="protein")
+output = task.get_output()
+output.render_interactive(representation="ribbon", add_backbone=True, selection="protein")
```

Comment:
this is also the example we have in the main readme right? we should probably change it, or add some information saying that the render works only on notebooks and requires the molecules_extra

Reply:
Yes, I will update it in another PR :)